index.js

```diff
@@ -173,23 +173,25 @@ export default class ClearvisioAppointmentBooker {
   async loadEyeExaminationProcesses({eyeExaminationProcessId}) {
     if (eyeExaminationProcessId) {
       try {
-        const  process = await api.get(this.store, `eye_examination_processes/${eyeExaminationProcessId}`);
-        return process
+        return [ await api.get(this.store, `eye_examination_processes/${eyeExaminationProcessId}`) ];
       } catch(error) {
-        if (error.code == 404){
+        if (error.code == 404) {
           this.store.dispatch('moduleState/set', 'error.missingConfiguredProcessId');
           return;
-        } 
+        } else {
+          return error;
```

Comment:
Megint egy if aminek egyik ágában van egy return, tehát akkor miért kell else ág külön blokkban eggyel mélyebbben. Másik kérdés, hogy egyébként is ezen `return error` helyett nem egy throw error-ra gondoltunk?